### PR TITLE
planner_cspace_msgs: add continuous movement mode to MoveWithTolerance action

### PR DESCRIPTION
**Pull request by nhatao**
This PR is required for https://github.com/at-wat/neonavigation/pull/652.

### CODE REVIEW

**Comment by at-wat** [comment]
## [37] PASSED on noetic

<details><summary>All tests passed</summary>


```
build/test_results/trajectory_tracker_msgs/gtest-test_path_with_velocity_conversion.xml: 4 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 5 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Comment by at-wat** [comment]
## [37] PASSED on melodic

<details><summary>All tests passed</summary>


```
build/test_results/trajectory_tracker_msgs/gtest-test_path_with_velocity_conversion.xml: 4 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 5 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Comment by at-wat** [comment]
## [44] PASSED on noetic

<details><summary>All tests passed</summary>


```
build/test_results/trajectory_tracker_msgs/gtest-test_path_with_velocity_conversion.xml: 4 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 5 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Comment by at-wat** [comment]
## [44] PASSED on melodic

<details><summary>All tests passed</summary>


```
build/test_results/trajectory_tracker_msgs/gtest-test_path_with_velocity_conversion.xml: 4 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 5 tests, 0 errors, 0 failures, 0 skipped
```
</details>

File: planner_cspace_msgs/action/MoveWithTolerance.action [hunk 1]
```diff
@@ -2,6 +2,7 @@ geometry_msgs/PoseStamped target_pose
 float64 goal_tolerance_lin
 float64 goal_tolerance_ang         # An angle tolerance to the goal point from the robot.
 float64 goal_tolerance_ang_finish  # An angle tolerance that the robot on the goal point should direct to a certain angle.
+bool is_continuous_movement_mode
```
Comment (at-wat) [inline planner_cspace_msgs/action/MoveWithTolerance.action:5]: I think `is_*` sounds like a status.
As it's a command, `continuous_movement_mode` or `continuous_movement` would be better.

**Comment by at-wat** [comment]
## [45] PASSED on melodic

<details><summary>All tests passed</summary>


```
build/test_results/trajectory_tracker_msgs/gtest-test_path_with_velocity_conversion.xml: 4 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 5 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Comment by at-wat** [comment]
## [45] PASSED on noetic

<details><summary>All tests passed</summary>


```
build/test_results/trajectory_tracker_msgs/gtest-test_path_with_velocity_conversion.xml: 4 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 5 tests, 0 errors, 0 failures, 0 skipped
```
</details>

File: planner_cspace_msgs/action/MoveWithTolerance.action (hunk 1 above)
Comment (nhatao) [reply planner_cspace_msgs/action/MoveWithTolerance.action]: Renamed to continuous_movement_mode.

File: planner_cspace_msgs/action/MoveWithTolerance.action [hunk 2]
```diff
@@ -2,7 +2,7 @@ geometry_msgs/PoseStamped target_pose
 float64 goal_tolerance_lin
 float64 goal_tolerance_ang         # An angle tolerance to the goal point from the robot.
 float64 goal_tolerance_ang_finish  # An angle tolerance that the robot on the goal point should direct to a certain angle.
-bool is_continuous_movement_mode
+bool continuous_movement_mode
```
Comment (at-wat) [inline planner_cspace_msgs/action/MoveWithTolerance.action:5]: Better to have a simple description of the mode

**Comment by at-wat** [comment]
## [46] PASSED on melodic

<details><summary>All tests passed</summary>


```
build/test_results/trajectory_tracker_msgs/gtest-test_path_with_velocity_conversion.xml: 4 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 5 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Comment by at-wat** [comment]
## [46] PASSED on noetic

<details><summary>All tests passed</summary>


```
build/test_results/trajectory_tracker_msgs/gtest-test_path_with_velocity_conversion.xml: 4 tests
build/test_results/trajectory_tracker_msgs/roslint-trajectory_tracker_msgs.xml: 1 tests
Summary: 5 tests, 0 errors, 0 failures, 0 skipped
```
</details>

**Review by at-wat — APPROVED** [review]
LGTM

**Pull request merged**